data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Sinai Syriac 2A, ff. 1-152",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
```

Comment:
not sure about the field name. But this is the ARK ID for the ms object, Sinai Syriac 2A.